README.md

```diff
@@ -2,6 +2,8 @@
 
 Exactly what you think this is. Go wild.
 
+Fork of [chrome-tabs](https://github.com/adamschwartz/chrome-tabs) to support UMD since original package has buggy AMD implementation.
```

Comment:
I meant to ask you to remove this but never got to it. Thank you for essentially doing that by creating #48.